src/Bindings/Plugin.h

```diff
@@ -111,12 +111,12 @@ class cPlugin
 	Command permissions have already been checked.
 	Returns true if command handled successfully
 	*/
-	virtual bool HandleCommand(const AStringVector & a_Split, cPlayer & a_Player) = 0;
+	virtual bool HandleCommand(const AStringVector & a_Split, cPlayer & a_Player, const AString & fullCommand) = 0;
```

Comment:
We use `a_ParamName` naming convention for function parameters, please do so as well, for consistency.